docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -566,6 +566,33 @@ Here one could find description of general HTTP properties applied across all th
 
 |===
 
+== Properties externalization
+
+. Declare https://en.wikipedia.org/wiki/Environment_variable[environment variable] containing desired property value and then 
+point the property to the declared environment variable:
++
+[source,properties]
+----
+bdd.variables.global.input-text=${MY_INPUT_TEXT}
+----
++
+[source,shell]
+----
+export MY_INPUT_TEXT=kekw
```

Comment:
we need to be more mindful in our examples

Reply:
:(